avaframe/ana1Tests/simiSolTest.py

```diff
@@ -737,7 +738,8 @@ def analyzeResults(particlesList, fieldsList, solSimi, fieldHeader, cfgSimi, out
             LMax error on flow velocity for saved time steps
 
     """
-    relTh = simDFrow['relTh']
+    # relTh = simDFrow['relTh']
```

Comment:
remove?

Reply:
I left this as simiSol needs more updates - there is a PR on this